packages/manager/modules/hub/src/components/billing-summary/template.html

```diff
@@ -20,13 +20,22 @@ <h3 data-translate="hub_billing_summary_title"></h3>
         </span>
         <span data-translate="hub_billing_summary_debt_null"></span>
     </p>
-    <p
-        data-ng-if="$ctrl.debt.dueAmount.value > 0"
-        data-translate="hub_billing_summary_debt"
-        data-translate-values="{
-         debt: $ctrl.formattedDebtPrice
-       }"
-    ></p>
+    <p data-ng-if="$ctrl.debt.dueAmount.value > 0">
+        <span
+            data-translate="hub_billing_summary_debt"
+            data-translate-values="{
+                debt: $ctrl.formattedDebtPrice
+              }"
+        >
+        </span>
+        <a
+            class="d-block"
+            data-ng-href="{{:: $ctrl.RedirectionService.getURL('debtPay') }}"
+            data-translate="hub_billing_summary_debt_pay"
+            target="_blank"
```

Comment:
```suggestion
            target="_blank"
            rel="noopener"
```
I'm wondering why not open in the same window ? this is the behaviour for all other links (see my bills, see my services etc)